en/firefox/privacy/book.ftl

```diff
@@ -0,0 +1,208 @@
+
+privacy-book-the-secure-site = The secure site:
+
+privacy-book-its-not-unusual = It’s not unusual to see addresses on the internet start with HTTP. This is a protocol that makes it possible to load web pages in the browser. Try, whenever possible, to use the encrypted option HTTPS instead. This is especially true if you are using a public WIFI: even if the connection itself is not encrypted, HTTPS will ensure that the data you transmit is encrypted nevertheless.
+
+privacy-book-email-accounts-and = Email accounts and online identities:
+
+privacy-book-your-email-address = Your email address doesn't necessarily reveal a lot about you. However, if you use the same address for a number of services and websites, you leave a trace. Your email address may become the center of your online identity, connected to all types of (browsing or account) data. Maybe you want to think about getting more than one for different purposes?
+
+privacy-book-resist-the-urge = Resist the urge to click:
+
+privacy-book-sometimes-its-easier = Sometimes it’s easier said than done – don’t click on dubious links! Most web users know to be particularly careful when it comes to e-mails that appear to come from their bank or an online retailer. However, when we receive links from friends and family it should be fine, right? Well, there are exceptions:
+
+privacy-book-from-time-to = From time to time social media accounts get hijacked and scammers send fraudulent links to the friends of the affected person. If you receive a message from a friend’s account that looks fishy or contains an odd, unknown link, give them a call to find out if they actually sent that note.
+
+privacy-book-its-also-possible = It’s also possible that nobody’s account got hijacked but your uncle still sends you a fraudulent link because he simply doesn’t know better. If you don’t know the site and prefer to be safe over being sorry, do a quick web search before clicking!
+
+privacy-book-look-at-the = Look at the bigger picture:
+
+privacy-book-we-already-talked = We already talked about how people tend to like convenience – remember? When it comes to information we consume, it might be nice to just see our attitude and expectations confirmed because it’s pleasant and calming. But does that make the information accurate? Do we get the big picture or just a rather small extract?
+
+privacy-book-you-can-of = <strong>You can, of course, have your favorites</strong> but check out different outlets from time to time to learn about different opinions and give other perspectives a chance.
+
+privacy-book-ask-yourselves-do = <strong>Ask yourself:</strong> Do the different outlets report on the issues you care about? And is the information shared more or less consistent or does it differ a lot? That’s the best way to battle filter bubbles and, ultimately, also filter bubbles and, ultimately, also misinformation.
```

Comment:
[Feedback from Peiying](https://github.com/mozilla-l10n/www-l10n/pull/127#discussion_r523254885):
> This sentence needs a rewrite. The three components are not in parallel: That’s the best way to **battle** filter bubbles and, ultimately, also **filter** bubbles and, ultimately, also misinformation. Battle and the 2nd filter are verbs. There needs to be a verb for misinformation.

Reply:
Good catch, seems like I hickup in editing. My bad. The sentence should be just easy:

"That’s the best way to battle filter bubbles and misinformation."